### ISSUE TITLE
EXIF orientation is not supported

### ISSUE THREAD

**Issue by fuelen** [gramps-project/Gramps.js#143]
https://github.com/recurser/exif-orientation-examples/blob/master/Portrait_4.jpg
if I upload the image above, the result is the following:
![зображення](https://user-images.githubusercontent.com/588762/203623223-ca0f5adc-42e6-4d97-aa11-ca805b1d7e1a.png)

**Comment by DavidMStraub** [comment]
We could add exif-js as dependency which would also allow us to auto-set the date for photos with EXIF

**Comment by DavidMStraub** [comment]
On the other hand ... at the moment the binary file is directly sent to the backend as-is using a `FileReader` object and I'm not sure how easy it is to do the rotation in the browser...

**Comment by DavidMStraub** [comment]
Actually ... this works for me in both Firefox and Chrome.

Which browser is this?

**Comment by fuelen** [comment]
Have you uploaded the picture?

Chromium 107.0.5304.110
In Firefox, I cannot open the localhost:5555 at all. I see blank screen.

**Comment by DavidMStraub** [comment]
>  In Firefox, I cannot open the localhost:5555 at all. I see blank screen.

Yes, that's expected when using the development server. It works with the built frontend.

> Have you uploaded the picture?

Oh, now I see it. I thought it was the preview before uploading.

**Comment by DavidMStraub** [comment]
Ok, now that I have finally understood, I realize it's not a frontend image: the thumbnailer needs to take Exif orientation into account. Moving to Web API.
